lib/websock.ex

```diff
@@ -81,8 +81,10 @@ defmodule WebSock do
   * `{:stop, reason :: term(), close_detail(), state()}`: Similar to the previous clause, but allows
     for the explicit setting of either a plain close code or a close code with a body to be sent to
     the client
+  * `{:stop, reason :: term(), close_detail(), state()}`: Similar to the previous clause, but allows
+    for the sending of one or more frames before sending the connection close frame to the client
   """
-  @callback handle_in({message(), opcode: data_opcode()}, state()) :: handle_result()
+  @callback handle_in({payload(), opcode: data_opcode()}, state()) :: handle_result()
```

Comment:
Just curious, can iodata actually occur here in a callback, or is it always a binary?

Reply:
Resolved.